main/exercise/question.class.php

```diff
@@ -2277,7 +2472,7 @@ public static function get_course_medias(
      *
      * @return int
      */
-    public static function get_count_course_medias($course_id)
+    public static function get_count_course_medias($courseId)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $table_question is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)